src/clj/game/cards/events.clj

```diff
@@ -874,6 +874,12 @@
    "Government Investigations"
    {:flags {:psi-prevent-spend (req 2)}}
 
+   "Guinea Pig"
+   {:msg "trash all cards in the grip and gain 10 [credits]"
+    :effect (req (doseq [c (:hand runner)]
+                   (trash state :runner c {:unpreventable true}))
```

Comment:
Does `trash` need to be `async`?

Reply:
Can't think of anything, but I've been seeing crashes in `Compile` and `Street Peddler` when their `trash` calls aren't `async`, so thought it might apply here as well.

Reply:
Not making it async since that will need some recursive trashing to trash cards one-by-one when they should be trashed simultaneously. If it causes issues / crashes I can revisit it.

Should be good to merge now.